app.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const app = express()
+
+const {
+    getAllTopics
+} = require('./controllers/controllers')
+
+app.use(express.json())
```

Comment:
Not needed at this stage, only add code as and when you need it to pass a test to ensure we don't have bloated codebases